magda-web-client/src/config.ts

```diff
@@ -17,7 +17,7 @@ const fallbackApiHost = "https://dev.magda.io/";
 
 const DEV_FEATURE_FLAGS = {
     cataloguing: true,
-    previewAddDataset: true
```

Comment:
probably don't change the default value here

Reply:
Done